kiwixbuild/versions.py

```diff
@@ -31,9 +31,9 @@
 
 release_versions = {
     'libzim': None, # Depends of base deps (was None)
-    'libkiwix': None, # Depends of libzim (was 0)
+    'libkiwix': 0, # Depends of libzim (was 0)
     'kiwix-tools': None, # Depends of libkiwix and libzim (was 2)
-    'zim-tools': 0, # Depends of libzim (was None)
+    'zim-tools': None, # Depends of libzim (was None)
     'kiwix-desktop': None # Depends of libkiwix and libzim (was 4)
```

Comment:
We should probably also build kiwix-tools and kiwix-desktop to have new binaries using the new libkiwix.

Reply:
New build is probably not possible.
New release could be done for kiwix-tools, but not urgency. I will do it later.

Reply:
Why not possible ?

Reply:
Indeed, this is a major version.